wrappers/s2i/nodejs/Makefile

```diff
@@ -0,0 +1,19 @@
+VERSION=0.1
+IMAGE_NAME = docker.io/seldonio/seldon-core-s2i-nodejs:${VERSION}
+
+SELDON_CORE_DIR=../../..
+
+
+.PHONY: build
+build:
+	docker build -t $(IMAGE_NAME) .
+
+push_to_dockerhub:
+	docker push $(IMAGE_NAME)
+
+
+.PHONY: test
+test:
```

Comment:
make test does not work:

```
make test
docker build -t docker.io/seldonio/seldon-core-s2i-nodejs:0.1-candidate .
Sending build context to Docker daemon   25.6kB
Step 1/9 : FROM node:latest
 ---> d0d12094f6ab
Step 2/9 : LABEL io.openshift.s2i.scripts-url="image:///s2i/bin"
 ---> Using cache
 ---> aaef67401575
Step 3/9 : RUN mkdir microservice
 ---> Using cache
 ---> 6cfa93b61811
Step 4/9 : WORKDIR /microservice
 ---> Using cache
 ---> 4dffc6d86792
Step 5/9 : COPY microservice.js /microservice
 ---> Using cache
 ---> 318f520dc0a0
Step 6/9 : COPY package.json /microservice
 ---> Using cache
 ---> 42dcbdf715b9
Step 7/9 : RUN npm install
 ---> Using cache
 ---> 9f165e8ab34b
Step 8/9 : COPY ./s2i/bin/ /s2i/bin
 ---> Using cache
 ---> 8f075578815b
Step 9/9 : EXPOSE 5000
 ---> Using cache
 ---> bd7b6e6f5275
Successfully built bd7b6e6f5275
Successfully tagged seldonio/seldon-core-s2i-nodejs:0.1-candidate
IMAGE_NAME=docker.io/seldonio/seldon-core-s2i-nodejs:0.1-candidate test/run
/bin/sh: 1: test/run: not found
Makefile:17: recipe for target 'test' failed

```

Reply:
The recipe failed because it couldn't find the appropriate run file within the test folder which is present now. So this should be working now. Please review. Thanks.